third_party/move/move-compiler-v2/transactional-tests/tests/control_flow/print_bytecode.exp

```diff
@@ -0,0 +1,31 @@
+comparison between v1 and v2 failed:
+= processed 2 tasks
+= 
+= task 0 'print-bytecode'. lines 1-4:
+- // Move bytecode v6
```

Comment:
Need to regenerate after rebase?

Reply:
Not quite, V1 and V2 bytecode differs so this test doesn't work well as a V1V2Comparison.  I've added a new subdir /no-v1-comparison/ to allow this test to be run standalone.